src/routes/__layout.svelte

```diff
@@ -18,10 +19,31 @@
         { title: "Benefits", url: "/", id: "#Benefits", onClick: scrollIntoView },
         { title: "FAQ", url: "/faq" }
     ];
+
+    const METADATA = {
+        title: "TLIP: Paperless trade between border agencies and traders",
+        description:
+            "An infrastructure based on blockchain for secure and trusted exchange of data and trade certificates, to speed up border processes and provide real time visibility of consignments.",
+        image: "/assets/metadata-image.png"
```

Comment:
lets better use this image from figma
![image](https://user-images.githubusercontent.com/3624944/173840629-916e6f51-5cd9-4935-867b-5ad2e27c1fb9.png)